### PR TITLE
docs: named exports

### PR DESCRIPTION
**Pull request by joasmi723**
Made the naming and structure consistent for components used in the docs, and made the all use named exports.

### CODE REVIEW

File: packages/react-magma-docs/src/pages/design/alerts.mdx [hunk 1]
```diff
@@ -6,10 +6,9 @@ order: 1
 
 import { Link } from 'gatsby';
 import './design-guidelines-styles.css';
-import DocsHeading from '../../components/docs-heading';
-import LeadParagraph from '../../components/lead-paragraph';
-
-import { Alert } from 'react-magma-dom';
+import { DocsHeading } from '../../components/DocsHeading';
+import { LeadParagraph } from '../../components/LeadParagraph';
+import { Alert, Paragraph, Heading } from 'react-magma-dom';
```
Comment (rcgillet) [inline packages/react-magma-docs/src/pages/design/alerts.mdx:11]:
```suggestion
import { Alert } from 'react-magma-dom';
```

**Review by rcgillet — CHANGES_REQUESTED** [review]

**Comment by aaronmaturen** [comment]
```react-magma-docs: error undefined failed
react-magma-docs: Can't resolve './src/components/Masthead' in '/home/runner/work/react-magma/react-magma/packages/react-magma-docs'
react-magma-docs: If you're trying to use a package make sure that './src/components/Masthead' is installed. If you're trying to use a local file make sure that the path is correct.
react-magma-docs: not finished Generating image thumbnails - 140.498s```

**Comment by joasmi723** [comment]
> ```
> react-magma-docs: Can't resolve './src/components/Masthead' in '/home/runner/work/react-magma/react-magma/packages/react-magma-docs'
> react-magma-docs: If you're trying to use a package make sure that './src/components/Masthead' is installed. If you're trying to use a local file make sure that the path is correct.
> react-magma-docs: not finished Generating image thumbnails - 140.498s```
> ```

I changed the folder name from `masthead` to `Masthead`, but it looks like that change isn't sticking.  Anyone have any ideas of how to fix that?

File: package-lock.json [hunk 2]
```diff
@@ -7185,7 +7185,7 @@
     },
     "confusing-browser-globals": {
       "version": "1.0.10",
-      "resolved": "https://registry.npmjs.org/confusing-browser-globals/-/confusing-browser-globals-1.0.10.tgz",
+      "resolved": "http://nexus.mindtap.corp.web/content/groups/npm.group/confusing-browser-globals/-/confusing-browser-globals-1.0.10.tgz",
```
[the remark this answers is not in the document]
Comment (joasmi723) [reply package-lock.json]: Fixed with Aaron's change

**Review by rcgillet — APPROVED** [review]

**Pull request merged**